fleet/lib/openvpn-db-pre/base/external-secret-openvpndb.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: external-secrets.io/v1
+kind: ExternalSecret
+metadata:
+  name: mariadb
+  namespace: openvpn-db
+spec:
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: onepassword
+  target:
+    name: mariadb
+    creationPolicy: Owner
+  data:
+    - secretKey: mariadb-root-password
```

Comment:
I think it would be better self-documenting to use separated 1pass items and k8s secrets here as the operator isn't force the use of a single k8s secret.

Ideally, the k8s secret name would match the 1pass item name and the 1pass field name would match the secret field name to remove any possibility for confusion.